stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/ChannelListItemListenerContainer.kt

```diff
@@ -0,0 +1,36 @@
+package io.getstream.chat.android.ui.channel.list.adapter.viewholder
+
+import io.getstream.chat.android.ui.channel.list.ChannelListView
+
+public class ChannelListItemListenerContainer(
```

Comment:
Perhaps just `ChannelListListenerContainer`, for a slightly simpler name? We could also make this an interface with an internal implementation, just so that nobody gets the idea of creating their own instances of it.

Alternatively, if we're not exposing it anywhere, we could make it entirely internal. The original `ListenerContainer` is only public because it's provided as a property in both `MessageViewHolderFactory` and `AttachmentViewHolderFactory`, but I see that `ChannelListItemViewHolderFactory` is just getting each individual listener as separate parameters.